### PR TITLE
Update ldap.md

### PR DESCRIPTION
**Pull request by byAdam**

### CODE REVIEW

File: docs/services/ldap.md [hunk 1]
```diff
@@ -2,10 +2,7 @@
 
 ## Current Situation
 
-- Primary LDAP Runs on Morpheus.internal
-- Secondary LDAP Runs on Deathray.internal, slaving Morpheus
-- Linux User machines (Minerva, Morpheus, Carbon, Murphy) Configured to try both
-  Servers
+- Primary LDAP Runs on daedalus.internal
```
Comment (m1cr0man) [inline docs/services/ldap.md:5]: It should be highlighted also that:
- icarus.internal is slaved to it
- Both are deployed via NixOS

**Review by m1cr0man — CHANGES_REQUESTED** [review]
Needs more info

File: docs/services/ldap.md (hunk 1 above)
Comment (theycallmemac) [reply docs/services/ldap.md:5]: Secondary server can be said to be icarus.internal so?

Comment (m1cr0man) [reply docs/services/ldap.md:5]: Well, no because it's not "secondary" from a fail over POV. It requires a manual DNS switch.

Comment (theycallmemac) [reply docs/services/ldap.md:5]: Has this changed since the weekend @m1cr0man ? Is it worth adding ldap2.internal?

**Comment by m1cr0man** [comment]
I have opened a new PR to rewrite most of these docs, closing this PR

**Pull request closed without merging**